### PR TITLE
add best practices

### PR DESCRIPTION
**Pull request by alextaaa**

### CODE REVIEW

File: .meteor/.finished-upgraders [hunk 1]
```diff
@@ -1,16 +0,0 @@
-# This file contains information which helps Meteor properly upgrade your
```
Comment (rodrigok) [inline .meteor/.finished-upgraders]: Why did you remove this file?

**Comment by rodrigok** [comment]
Hi @alextaaa, can you explain why you did this changes?

Thanks

**Comment by alextaaa** [comment]
Hi @rodrigok this file not required for meteor app

**Comment by rodrigok** [comment]
@alextaaa Can send us some docs where we can confirm this?

As far as I know this file is important.

**Comment by alextaaa** [comment]
So, can you prove the info `this file is important` asap ? ;)

**Comment by rodrigok** [comment]
The file is generated by Meteor and used to process migrations between version upgrades, so it is important. There is no need to remove it. If you can list some benefit we can discuss this.

**Comment by alextaaa** [comment]
But, it's only your opinion , I think you show links to the official documentation or invite core team guys ; )

**Comment by rodrigok** [comment]
@RocketChat/core Opinions here?

**Comment by engelgabriel** [comment]
Hi @alextaaa, I think the message added by MDG to top of the file is pretty clear:

> This file contains information which helps Meteor properly upgrade your app when you run 'meteor update'. You should check it into version control with your project.

I don't see a reason to not do that. If they didn't want this file to be included, they'd have added to the default `.meteor/.gitignore` file, together with `dev_bundle` and `local`.

File: package.json [hunk 2]
```diff
@@ -34,6 +34,10 @@
     {
       "name": "Sing Li",
       "email": "sing.li@rocket.chat"
+    },
+    {
+      "name": "Alex Terpyakov",
+      "email": "terpyakov@gmail.com"
```
Comment (engelgabriel) [inline package.json]: Thanks for your interest in joining the core team, but we only allow people with considerable contributions to become part of the core team. I hope you understand.

File: .meteor/packages [hunk 3]
```diff
@@ -132,7 +133,8 @@ konecty:multiple-instances-status
 konecty:nrr
 konecty:user-presence
 
-chrismbeckett:toastr
+lamatyus:toastr
```
Comment (engelgabriel) [inline .meteor/packages]: I understand you are the author of this package, is there a GitHub repo for it?

File: package.json (hunk 2 above)
Comment (alextaaa) [inline package.json]: Thanks for explain ;)

**Comment by marceloschmidt** [comment]
Can we close this PR?

**Comment by geekgonecrazy** [comment]
@marceloschmidt I don't think there is any further action on this one.  At least that i can tell :)

**Comment by rodrigok** [comment]
@alextaaa please reopen this PR if you want to do more changes
